includes/reports/overview/list-table/class-sensei-reports-overview-list-table-courses.php

```diff
@@ -182,7 +194,9 @@ protected function get_row_data( $item ) {
 
 		// Output course data.
 		/** This filter is documented in wp-includes/post-template.php */
-		$course_title = apply_filters( 'the_title', $item->post_title, $item->ID ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals
+		$course_title   = apply_filters( 'the_title', $item->post_title, $item->ID ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals
```

Comment:
Looks like we don't need this phpcs comment here...
Could you try to remove it and see if it passes php-linter?
```suggestion
		$course_title   = apply_filters( 'the_title', $item->post_title, $item->ID );
```

Reply:
<img width="635" alt="Screenshot 2023-06-15 at 16 46 18" src="https://github.com/Automattic/sensei/assets/38718/6a432744-8aca-4793-a2ad-1d410583c20f">